packages/async-storage/webui/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@dev-plugins/async-storage-webui",
+  "description": "The frontend webui for @dev-plugins/async-storage",
+  "private": true,
+  "version": "0.0.5",
+  "main": "index.js",
+  "scripts": {
+    "start": "expo start -w"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "dependencies": {},
+  "devDependencies": {
+    "@ant-design/icons": "^5.2.6",
+    "@babel/core": "^7.20.0",
+    "@react-native-async-storage/async-storage": "^1.21.0",
+    "@types/react": "~18.2.45",
+    "antd": "^5.10.2",
+    "expo": "50.0.0-preview.7",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "@microlink/react-json-view": "^1.23.0",
```

Comment:
is that `@microlink/react-json-view` still maintained so you using it here? do you recommend to migrate other react-json-view to `@microlink/react-json-view`?

Reply:
See the [message](https://github.com/mac-s-g/react-json-view?tab=readme-ov-file#%EF%B8%8F-this-project-is-no-longer-being-maintained-%EF%B8%8F) from the original project. Looks like there's also uiwjs's [react-json-view](https://github.com/uiwjs/react-json-view) which I didn't really look into. Not sure how well maintained it is, but it seems stable at the very least

Reply:
wow just check the [comparison in react-json-view](https://github.com/uiwjs/react-json-view?tab=readme-ov-file#size-and-dependencies) and find the [react-inspector](https://github.com/storybookjs/react-inspector). that's more promising to me, i will try to migrate react-inspector for other plugins.

Reply:
Agreed, I'll switch this PR over to it

Reply:
Looking more at it I'm not actually sure that react-inspector is a good fit as it doesn't seem to really support editing. I think as far as this plugin goes sticking with `@microlink/react-json-view` would be best, and maybe switch over to [react-json-view](https://uiwjs.github.io/react-json-view/) if there are any issues (and once it's 2.0 comes out of alpha). But the usage is pretty limited so I don't really see any reason it should break (outside of a React update I suppose)